eva/readers/abstract_reader.py

```diff
@@ -52,7 +52,10 @@ def read(self) -> Iterator[Batch]:
         row_size = None
         for data in self._read():
             if row_size is None:
-                row_size = data['data'].nbytes
+                if 'data' in data:
```

Comment:
We are assuming that we always have an attribute 'data' in the result of SELECT queries. But this may not be the case. For example: SELECT queries on meta table. I had to add this change, to avoid errors.

Reply:
This is fixed in @gaurav274 's latest PR